src/ripple/protocol/STVector256.h

```diff
@@ -144,6 +144,18 @@ class STVector256
         return mValue;
     }
 
+    std::vector<uint256>::iterator
+    insert(std::vector<uint256>::const_iterator pos, uint256 const& value)
+    {
+        return mValue.insert(pos, value);
+    }
+
+    std::vector<uint256>::iterator
+    insert(std::vector<uint256>::const_iterator pos, uint256&& value)
+    {
+        return mValue.insert(pos, value);
```

Comment:
`uint256` is not move friendly, so I don't think we need this function. However, if we do want to keep it, we should still `std::move(value)` here.